app/utils.py

```diff
@@ -154,3 +154,14 @@ def purge_nulls_from_data(data):
 def get_request_page_questions():
     json_payload = get_json_from_request()
     return json_payload.get('page_questions', [])
+
+
+def validate_user_can_access_direct_award_project_or_403(user_id, project_id):
```

Comment:
It's worth checking what other write endpoints do, but I think if the user_id is not stored or used for integrity checks and is only present for authorization then we follow the same rule of checking in the frontend.